cmd/ci-secret-mirroring/README.md

```diff
@@ -0,0 +1,31 @@
+# CI-Secret-Mirroring
+
+This tool extends the [openshift/ci-secret-mirroring-controller](https://github.com/openshift/ci-secret-mirroring-controller)
```

Comment:
We can use `openshift/ci-secret-mirroring-controller` for the src if the following plan gets approved.